test/test_things.js

```diff
@@ -1,18 +1,200 @@
-import app           from './server';
 import should        from 'should';
 import supertest     from 'supertest';
+import app           from './server';
+import db           from '../src/models/db';
 
 const server = supertest.agent(app);
+const resource = 'Things';
+
+let defaultThingObject = {};
+let responseList = {};
+let id = null;
 
 describe('Things API', () => {
-  it('should exist', done => {
-    server.get('/Things')
-// XXX      .expect('Content-Type', /json/)
-      .expect(200)
-      .end((err, res) => {
-        should.not.exist(err);
-        res.status.should.be.equal(200);
-        done();
+  beforeEach(() => {
+    defaultThingObject = {
+      'name': 'name',
+      'description': 'camping lantern',
+      'properties': {
+        'property1': 'it’s waterproof',
+        'property2': 'it glows in the dark',
+        'property3': 'it repels insects'
+      }
+    };
+    responseList = {
+      '@iot.count': 1,
+      'value': [defaultThingObject]
+    };
+  });
+
+  describe('GET Things API', () => {
+    beforeEach(done => {
+      db().then(models => {
+        models[resource].create(defaultThingObject).then((createdThing) => {
+          id = createdThing.id;
+          responseList.value[0]['@iot.id'] = id;
+          responseList.value[0]['@iot.selfLink'] = '/' + resource + '(' + id + ')';
+          done();
+        });
+      });
+    });
+
+    afterEach((done) => {
+      db().then(models => {
+        models[resource].findById(id).then(thing => {
+          thing.destroy().then(() => {
+            done();
+          })
+        });
+      });
+    });
+
+    it('should return a list of Things', done => {
+      server.get('/' + resource)
+// XXX      .expect('Content-Type', /json/)
```

Comment:
You should be able to remove the comments for this check in all tests now that you've implemented the Things router